icons/jar.svg

```diff
@@ -0,0 +1,4 @@
+<?xml version="1.0" encoding="UTF-8" standalone="no"?>
+<svg xmlns="http://www.w3.org/2000/svg" viewBox="0 0 24 24">
+  <path d="M7.4 1.7a5.7 5.7 0 0 0-5.7 5.7v9.2c0 3.1 2.6 5.7 5.7 5.7h9.2c3.1 0 5.7-2.6 5.7-5.7V7.4c0-3.1-2.6-5.7-5.7-5.7H7.4zM6 6.2h12.2c1 0 1.7.7 1.7 1.6v2.4a1.7 1.6 0 0 1-1.7 1.5h-1.7v1.8a3 2.9 0 0 1-3 2.9H9a3 2.9 0 0 1-3-3V6.3zm10.5 1.6v2.4h1.7V7.8h-1.7zm-12.2 10h15.2v1.6H4.3v-1.6z" fill="#f44336"/>
```

Comment:
Maybe make `jar` consistent with `zip` icon?

```suggestion
  <path d="M 5 3 C 3.9 3 3 3.9 3 5 L 3 19 C 3 20.1 3.9 21 5 21 L 19 21 C 20.1 21 21 20.1 21 19 L 21 5 C 21 3.9 20.1 3 19 3 L 5 3 z M 6 7 L 16 7 C 17 7 18 8 18 9 L 18 11 C 18 12 17 13 16 13 L 14 13 L 14 15 C 14 16 13 17 12 17 L 8 17 C 7 17 6 16 6 15 L 6 7 z M 14 9 L 14 11 L 16 11 L 16 9 L 14 9 Z" fill="#f44336"/>
```

Reply:
Comparison (on different DPI):
![prev](https://user-images.githubusercontent.com/49621788/121267577-6924e600-c88a-11eb-8fdd-e7a2518ffbad.png)

Reply:
I was expecting it to perform much better at lower DPIs. I like the size of the mug in the old icon, but I don't like how round the outside is, and the bar is too wide also.

How about:
```suggestion
  <path d="m5 3c-1.1 0-2 0.9-2 2v14c0 1.1 0.9 2 2 2h14c1.1 0 2-0.9 2-2v-14c0-1.1-0.9-2-2-2h-14zm2 3h10c1.1 0 2 0.9 2 2v2c0 1.1-0.9 2-2 2h-2c0 3-3 3-4 3s-4 0-4-3v-6zm8 2v2h2v-2h-2zm-10 8h12v2h-12v-2z " fill="#f44336"/>
```